recipe/meta.yaml

```diff
@@ -93,10 +91,12 @@ outputs:
         - python
         - pip
         - cython
+        - libhwloc
```

Comment:
As of PR ( https://github.com/rapidsai/ucx-py/pull/342 ), `ucx-py` now depends on `libhwloc`. So we add it to `host` and `run`.

This is not needed for `ucx` itself though.

Reply:
Also for reference, we may want to merge after PR ( https://github.com/conda-forge/libhwloc-feedstock/pull/20 ) is merged and packages released (if we want the latest `libhwloc` release).